host_vars/dev.gvl.org.au.yml

```diff
@@ -357,14 +357,14 @@ singularity_default_container_id: "/cvmfs/singularity.galaxyproject.org/all/pyth
 
 # multisite stuff
 
-galaxy_subsite_base_domain: "dev.usegalaxy.org.au"
+galaxy_subsite_base_domain: "dev.gvl.org.au"
 galaxy_subsite_default_welcome: "https://site.usegalaxy.org.au"
 galaxy_subsite_dir: /mnt/galaxy/subsites
 #galaxy_subsite_base_css: '#masthead { background-color: #003399;}'
 galaxy_subsites:
   - name: genome
     brand: Genomes lab
-    iframe: "https://dev-site.usegalaxy.org.au/landing/genome"
+    iframe: "https://dev-site.gvl.org.au/landing/genome"
```

Comment:
this line probably does not need to change

Reply:
dev-site.usegalaxy.org.au is not the same as site.usegalaxy.org.au, which is referenced in line 361. Once dev*.usegalaxy.org.au DNS entries are removed and only dev*.gvl.org.au DNS entries remain, then dev-site.usegalaxy.org.au won't exist.